### DISCUSSION TITLE
Timestamp format

### DISCUSSION THREAD

**Discussion by 8vp** [COVESA/dlt-daemon#443] [Q&A] [answered]
From the spec, I see that there are 2 possible ways to configure timestamps. One is built in and the other can be user defined. Both 4 bytes. Not able to understand the format of these default timestamp. It is like some form of epoch? Classic AUTOSAR DLT version 21_11 uses 9 bytes for timestamp (Unix Epoch, additionally with nanosec resolution). How is the Covesa timestamp compatible to this standard? Please advise.

**Comment by michael-methner** [comment, marked as answer]
Hello @8vp ,
do you mean:
https://github.com/COVESA/dlt-daemon/blob/master/doc/dlt_for_developers.md#using-custom-timestamps
it is:
uint32_t timestamp = 1234567; /* uptime in 0.1 milliseconds */

this is per "Log and Trace Protocol Specification" in Autosar version 1.2.0 (check for TMSP)
